addons/account/models/company.py

```diff
@@ -217,6 +217,7 @@ class ResCompany(models.Model):
         readonly=False,
         help="The country to use the tax reports from for this company")
     account_fiscal_country_group_codes = fields.Json(compute="_compute_account_fiscal_country_group_codes")
+    country_code = fields.Char(related='account_fiscal_country_id.code', readonly=True)
```

Comment:
```suggestion
    country_code = fields.Char(related='account_fiscal_country_id.code')
```
readonly is the default True for related fields and shouldn't we make consistency by adding depends parameter as well with base?

Reply:
to change the depends to fiscal_country_id?  Seems a good idea.